metabase/metabase
# make series style settings tab first

[Slack convo](https://metaboat.slack.com/archives/C064QMXEV9N/p1722970957608869)

### Description

This PR makes the series style settings tab first.

### How to verify

- New Orders -> Count by Created At -> it should show a line chart
- Open "Count" series settings
- Ensure it immediately shows the line style settings instead of formatting settings

### Demo

<img width="894" alt="Screenshot 2024-08-06 at 3 45 54 PM" src="https://github.com/user-attachments/assets/a52b6d59-b0cb-4d42-81ef-ca0f35ad2ccb">

### Checklist
